src/Formik.tsx

```diff
@@ -411,13 +389,13 @@ export function useFormik<Values = object>({
       // what is called when using validateForm.
 
       if (
-        fields.current !== null &&
-        fields.current[name] &&
-        fields.current[name].validate &&
-        isFunction(fields.current[name].validate)
+        fieldRegistry.current !== null &&
+        fieldRegistry.current[name] &&
```

Comment:
I think it's safe to omit this check, the non-truthy value wouldn't pass through `typeof === 'function'` anyway.

Reply:
Yep